### ISSUE TITLE
(Feature Request): Make so that the like counter is only the likes from the addon users

### ISSUE THREAD

**Issue by Pdzly** [Anarios/return-youtube-dislike#151]
### Extension or Userscript?

Extension

### Request or suggest a new feature!

Replace the like count with the "real" good/bad ratio
Because then we have 100% the correct look on the video and not a 2k upvotes and just 200 downvotes but the downvotes are just from the addon but the likes are from any source and if the like count is from the addon too so its like 100 likes and 200 dislikes and i would rather think they really plan to remove the like button are on its way too. 

### Ways to implement this!

Just the same principle like the dislike counter

### Can you work on this?

- [ ] Yes
- [X] No

**Comment by mattphelps-8451** [comment]
I disagree with this. Likes are reliable and come straight from Youtube. The unknown variable is dislikes, not likes. We should estimate dislikes as best as possible.

**Comment by Panzer04** [comment]
I guess the idea is that the add on only captures a fraction of dislikes compared to all likes. This means 1000 likes and 1000 dislikes might display as 1000/100 (because, generously, only 10% use the add on)

Effectiveness depends on how good the adjustment algorithm is for dislikes, if that works well this isn’t super important. I suspect the add on will want to record likes anyway to assist in estimating the real counts.

**Comment by llagerlof** [comment]
I like the idea as an option in configuration, like:

- [x]  Youtube likes / Plugin dislikes estimates (as it will be after December 13th, 2021)
- [ ]  Plugin user's likes / Plugin users's dislikes

**Comment by rewrib** [comment]
the youtube likes should be visible somewhere, maybe in brackets after the plugin likes

**Comment by niwla23** [comment]
related: #109
